types/node-forge/index.d.ts

```diff
@@ -16,7 +16,7 @@
 //                 Dmitry Avezov       <https://github.com/avezov>
 //                 Jose Fuentes        <https://github.com/j-fuentes>
 //                 Anya Reyes          <https://github.com/darkade>
-//                 Tino                <https://github.com/tino-247>
+//                 Tino                
```

Comment:
```suggestion          
```

If you don't want to be on the author list, the right change here is to just delete the whole line; credit is handled via git history, so this list only controls PR review automation.